composer.json

```diff
@@ -92,8 +92,9 @@
         "behat/mink-selenium2-driver": "~1.2",
         "behat/mink": "~1.6",
         "akeneo/phpspec-skip-example-extension": "~1.2",
-        "coduo/php-matcher": "~1.0",
         "lakion/mink-debug-extension": "^1.0.1",
+        "lakion/api-test-case": "dev-master",
```

Comment:
We should set a branch alias on ApiTestCase's composer.json and require here: ^1.0@dev.